src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/TagletWriterImpl.java

```diff
@@ -388,7 +386,7 @@ public Content seeTagOutput(Element holder, List<? extends SeeTree> seeTags) {
         }
         // Use a different style if any link label is longer than 30 chars or contains commas.
```

Comment:
It's a good opportunity to be DRY; I suggest:
    // Use a different style if any link label is long or contains commas.

Reply:
Maybe in the next round of cleanup ...